### PR TITLE
[Outlook] (item multi-select) Create walkthrough

### PR DESCRIPTION
**Pull request by samantharamon**
Other item multi-select PRs:
- https://github.com/OfficeDev/office-js-docs-reference/pull/1415
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62580

### CODE REVIEW

File: docs/outlook/item-multi-select.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+          }
+    
+          console.log("Event handler added.");
+        });
+    }
+    ```
+
+## Retrieve the subject line of selected messages
+
+Now that you've registered an event handler, you then call the [getSelectedItemsAsync](/javascript/api/outlook/office.mailbox?view=outlook-js-preview#outlook-office-mailbox-getSelectedItemsAsync-member(1)&preserve-view=true) method to retrieve the subject line of the selected messages and log them to the task pane. The `getSelectedItemsAsync` method can also be used to get other message properties, such as the item ID, item type (Message is the only supported type at this time), and item mode (Read or Compose).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/item-multi-select.md:158]:
```suggestion
Now that you've registered an event handler, you then call the [getSelectedItemsAsync](/javascript/api/outlook/office.mailbox?view=outlook-js-preview#outlook-office-mailbox-getSelectedItemsAsync-member(1)&preserve-view=true) method to retrieve the subject line of the selected messages and log them to the task pane. The `getSelectedItemsAsync` method can also be used to get other message properties, such as the item ID, item type (`Message` is the only supported type at this time), and item mode (`Read` or `Compose`).
```

File: docs/outlook/item-multi-select.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+- An add-in must implement a task pane in order to detect the `SelectedItemsChanged` event.
+- The [Reading Pane](https://support.microsoft.com/office/use-and-configure-the-reading-pane-to-preview-messages-2fd687ed-7fc4-4ae3-8eab-9f9b8c6d53f0) in Outlook must be enabled.
+- A maximum of 100 messages can be selected at a time.
+
+> [!NOTE]
+> Meeting invites and responses are considered messages, not appointments, and can therefore be included in a selection.
+
+### Multi-select in conversations
+
+Item multi-select supports [Conversations view](https://support.microsoft.com/office/view-email-messages-by-conversation-0eeec76c-f59b-4834-98e6-05cfdfa9fb07) whether it's enabled on your mailbox or on specific folders. The following table describes expected behaviors when conversations are expanded or collapsed, when the conversation header is selected, and when conversation messages are located in a different folder from the one currently in view.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/item-multi-select.md:222]: Better enables users to hit their localized version
```suggestion
Item multi-select supports [Conversations view](https://support.microsoft.com/office/0eeec76c-f59b-4834-98e6-05cfdfa9fb07) whether it's enabled on your mailbox or on specific folders. The following table describes expected behaviors when conversations are expanded or collapsed, when the conversation header is selected, and when conversation messages are located in a different folder from the one currently in view.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

File: docs/outlook/item-multi-select.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+title: Activate your Outlook add-in on multiple messages (preview)
+description: Learn how to activate your Outlook add-in when multiple messages are selected.
+ms.topic: article
+ms.date: 10/05/2022
+ms.localizationpriority: medium
+---
+
+# Activate your Outlook add-in on multiple messages (preview)
+
+With the item multi-select feature, your Outlook add-in can now activate and perform operations on multiple selected messages in one go. Certain operations, such as uploading messages to your CRM system or categorizing numerous items, can now be easily completed with a single click.
```
[the remark this answers is not in the document]
Comment (lindalu-MSFT) [reply docs/outlook/item-multi-select.md:11]: e.g.
... uploading messages to your Customer Relationship Management (CRM) system or...

**Review by lindalu-MSFT — APPROVED** [review]
Not really a review, just a comment! Looks like Elizabeth did a good job on her review! Plus, you've caught on quickly and learned MSFT voice and style.

File: docs/outlook/item-multi-select.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+- A maximum of 100 messages can be selected at a time.
+
+> [!NOTE]
+> Meeting invites and responses are considered messages, not appointments, and can therefore be included in a selection.
+
+### Multi-select in conversations
+
+Item multi-select supports [Conversations view](https://support.microsoft.com/office/view-email-messages-by-conversation-0eeec76c-f59b-4834-98e6-05cfdfa9fb07) whether it's enabled on your mailbox or on specific folders. The following table describes expected behaviors when conversations are expanded or collapsed, when the conversation header is selected, and when conversation messages are located in a different folder from the one currently in view.
+
+||Expanded conversation view|Collapsed conversation view|
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/outlook/item-multi-select.md]: Thanks, @lindalu-MSFT! I added "Selection" as the heading.

File: docs/outlook/item-multi-select.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+To preview the multi-select feature, install Outlook on Windows, starting with Version 2209 (Build 15629.20110). Once installed, join the [Office Insider program](https://insider.office.com/join/windows) and select the **Beta Channel** option to access Office beta builds.
+
+## Set up your environment
+
+Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator) to create an add-in project with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
+
+## Configure the manifest
+
+To enable your add-in to activate on multiple selected messages, you must add the [SupportsMultiSelect](/javascript/api/manifest/action?view=outlook-js-preview#supportsmultiselect&preserve-view=true) child element to the **\<Action\>** element and set its value to `true`. As item multi-select only supports messages at this time, the **\<ExtensionPoint\>** element's `xsi:type` attribute value must be set to `MessageReadCommandSurface` or `MessageComposeCommandSurface`.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/item-multi-select.md:28]: I think appending the second param breaks the bookmark/anchor... Here and other instances
```suggestion
To enable your add-in to activate on multiple selected messages, you must add the [SupportsMultiSelect](/javascript/api/manifest/action?view=outlook-js-preview&preserve-view=true#supportsmultiselect) child element to the **\<Action\>** element and set its value to `true`. As item multi-select only supports messages at this time, the **\<ExtensionPoint\>** element's `xsi:type` attribute value must be set to `MessageReadCommandSurface` or `MessageComposeCommandSurface`.
```

**Pull request merged**